tests/CMakeLists.txt

```diff
@@ -1,13 +1,53 @@
 # This is the CMake file for the tests directory of NCEPLIBS-ncio.
 
+# Syntax:  add_mpi_test (<TESTNAME>
+#                        EXECUTABLE <command>
+#                        ARGUMENTS <arg1> <arg2> ...
+#                        NUMPROCS <num_procs>
+#                        TIMEOUT <timeout>)
+function (add_mpi_test TESTNAME)
```

Comment:
Woudlnt' it be best to keep this in a separate cmake file, because many project will use it?

Reply:
Sure, we can do that.